Project.toml

```diff
@@ -34,7 +35,7 @@ OrdinaryDiffEq = "5, 6"
 QuantumOpticsBase = "0.3, 0.4, 0.5"
 RecursiveArrayTools = "2, 3"
 Reexport = "0.2, 1.0"
-StochasticDiffEq = "6"
+StochasticDiffEq = "6, 6.68.0"
```

Comment:
this should probably just be one single entry `6.68` if you want to refuse `6.67`